src/types/GitHubContext.Types.ps1xml

```diff
@@ -1,27 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Types>
     <Type>
-        <Name>UserGitHubContext</Name>
+        <n>UserGitHubContext</n>
```

Comment:
Change this back to "Name". In general, keep the correct schema for the XML file for types.ps1xml.